depthai_ros_driver/src/pipelines/mono_camera_control_pipeline.cpp

```diff
@@ -0,0 +1,67 @@
+#include <ros/console.h>
+
+#include <pluginlib/class_list_macros.h>
+#include <depthai_ros_driver/pipeline.hpp>
+
+#include <depthai/pipeline/node/MonoCamera.hpp>
+#include <depthai/pipeline/node/ImageManip.hpp>
+#include <depthai/pipeline/node/XLinkIn.hpp>
+#include <depthai/pipeline/node/XLinkOut.hpp>
```

Comment:
What about just including pipeline/nodes.hpp?

Reply:
are there any criteria if we include the whole `nodes.hpp` or independent headers?
I felt including necessary headers one by one is not a pain for this numbers.